chutney/action-impl/src/main/java/com/chutneytesting/action/function/MathFunctions.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.action.function;
+
+import com.chutneytesting.action.spi.SpelFunction;
+
+public class MathFunctions {
+
+    @SpelFunction
```

Comment:
Please add unit test.

As it will be very simple unit test, I think you could use @Forall fril jqwik to test it

Reply:
use @ParameterizedTest to check different input type